dmscripts/helpers/supplier_data_helpers.py

```diff
@@ -392,12 +390,3 @@ def unsuspend_suspended_supplier_services(record, suspending_user, client, logge
             logger.info(f"[DRY RUN] Would unsuspend service {service_id} for supplier {supplier_id}")
         else:
             client.update_service_status(service_id, new_service_status, "Unsuspend services helper")
-
-
-def get_email_addresses_for_supplier(api_client: DataAPIClient, supplier_id: int) -> List[str]:
```

Comment:
I could leave this here, since it's a function that we might want to use in other scripts

Reply:
Arguably my fault for including generic improvements within G12 PRs :slightly_smiling_face:

Not difficult to re-instate if we discover we need/want this in future.